### ISSUE TITLE
[Patreon goal] Provide fullscreen view controllable with joypad

### ISSUE THREAD

**Issue by tsiegleauq** [lutris/lutris#303]
Hi.
I thought it was cool to use lutis as a (better) replacement for the Steam Big Picture mode,
So we would need a way for Game-pad input and maybe a more - TV oprimzed overview alternative.

Do you think something like is is possible?

Being able to have Game-pad input and running Steam games and Emulator-games from the same interface would be incredible awesome!

**Comment by RobLoach** [comment]
Two different methods I've been using:

1. From [Kodi](https://kodi.tv/), the [Lutris Kodi Addon](https://github.com/robloach/script.lutris) will launch games for you.
2. You can also [add Lutris games to Steam](https://github.com/lutris/lutris/wiki/Steam-Integration) to have them launch from Big Picture. This is not automated though, which would be a nice thing to have.

Having gamepad support directly in the Lutris application would be pretty cool too! Thoughts?

**Comment by strycore** [comment]
Joypad support will be implemented but don't expect it soon, there are lots of features that have higher priority on the roadmap. Of course, if someone want to jump in and submit a pull request, that would be much quicker!

**Comment by billywade** [comment]
@strycore is this something that could be added into the existing PyQt or is this something that would need to be built from the ground up using a different graphics library/cannibalized from an existing UI like Bottlenose or EmulationStation?

**Comment by strycore** [comment]
The current plans are to use GtkSceneKit

**Comment by billywade** [comment]
So we're mostly just waiting on the Gnome foundation and somebody to integrate it into PyGTK?

**Comment by strycore** [comment]
No, pygtk doesnt exist anymore. The moment this is available in Gtk, we can start using it

**Comment by billywade** [comment]
Did not notice that pygtk died _eight years ago_. Sounds good
